src/ucp/wireup/wireup_cm.c

```diff
@@ -118,6 +118,20 @@ static int ucp_cm_client_try_fallback_cms(ucp_ep_h ep)
     return 1;
 }
 
+static ucp_rsc_index_t
+ucp_cm_ep_tl_bitmap_get_dev_idx(ucp_worker_h worker, uint64_t tl_bitmap)
+{   
+    ucp_rsc_index_t rsc_index;
+    ucp_rsc_index_t dev_index;
+
+    ucs_assert(tl_bitmap != 0);
+    rsc_index = ucs_ffs64_safe(tl_bitmap);
+    ucs_assert(rsc_index != UCP_NULL_RESOURCE);
```

Comment:
this assert looks odd here, i'd remove

Reply:
yes, need to compare with `UCP_MAX_RESOURCES` that is `64`